### PR TITLE
Update link to proxy docs

### PR DESCRIPTION
**Pull request by miiu96**
Updated the link to the elrond proxy documentation.

### CODE REVIEW

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
please update the readme also. endpoints like esdt handling are missing

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ For more details, go to [docs.elrond.com](https://docs.elrond.com/sdk-and-tools/
 - `/address/:address/shard`   (GET) --> returns the shard of an :address based on current proxy's configuration.
 - `/address/:address/storage/:key`   (GET) --> returns the value for a given key for an account.
 - `/address/:address/transactions` (GET) --> returns the transactions stored in indexer for a given :address.
+- `/address/:address/esdt` (GET) --> returns the account's ESDT tokens list for the given :address.
+- `/address/:address/esdt/:tokenIdentifier` (GET) --> return the account's balance of a given :address and ESDT token
```
Comment (bogdan-rosianu) [inline README.md:20]: maybe `returns the token data for a given :address, such as balance or properties`

Comment (miiu96) [reply README.md]: done.

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ For more details, go to [docs.elrond.com](https://docs.elrond.com/sdk-and-tools/
 - `/address/:address/storage/:key`   (GET) --> returns the value for a given key for an account.
 - `/address/:address/transactions` (GET) --> returns the transactions stored in indexer for a given :address.
 - `/address/:address/esdt` (GET) --> returns the account's ESDT tokens list for the given :address.
-- `/address/:address/esdt/:tokenIdentifier` (GET) --> return the account's balance of a given :address and ESDT token
+- `/address/:address/esdt/:tokenIdentifier` (GET) --> return the token data for a given :address and ESDT token, such as balance and properties
```
Comment (bogdan-rosianu) [inline README.md:20]: returns*

**Review by bogdan-rosianu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**